monorepo/services/server/src/graphql/resolvers/lead-report.js

```diff
@@ -97,7 +97,18 @@ module.exports = {
         },
       });
       const results = await OmedaEmailClick.aggregate(pipeline);
-      return emailReportService.buildEmailMetrics({ results, sort, deploymentEntities });
+      const metricsObj = await emailReportService.buildEmailMetrics({
```

Comment:
Since you aren't returning this directly, you'll need to await it

Reply:
I believe this is already correct? I could be missing something however.